virtcontainers/qemu_amd64.go

```diff
@@ -93,6 +70,42 @@ func newQemuArch(config HypervisorConfig) qemuArch {
 		factory = true
 	}
 
+	var q35QemuMachineOptions string
+
+	if config.IOMMU {
+		kernelParams = append(kernelParams,
```

Comment:
I would add a little bit of logging here so that a log captures which options were sent to the kernel, etc. Probably `Info` level.

Reply:
Adding a comment here will only log amd64 kernel params.
On every execution we will get a Info level log with all qemu params (including kernel params) in:
https://github.com/intel/govmm/blob/7efaf0b1cde3883a3f38e656c2223f38b9086469/qemu/qemu.go#L2591

Is that enough in your opinion or should we log here as well just the kernel params?